var/spack/repos/builtin/packages/pdt/package.py

```diff
@@ -17,8 +17,9 @@ class Pdt(AutotoolsPackage):
 
     """
     homepage = "https://www.cs.uoregon.edu/research/pdt/home.php"
-    url      = "http://www.cs.uoregon.edu/research/paracomp/pdtoolkit/Download/pdtoolkit-3.22.1.tar.gz"
+    url      = "http://www.cs.uoregon.edu/research/paracomp/pdtoolkit/Download/pdtoolkit-3.25.1.tar.gz"
```

Comment:
Are these other changes intentional, or did they just get mixed up in this PR?

Reply:
It's not clear to me why there are 11 commits in this PR.  Looks like things did get mixed up, or the PR needs to be cleaned up.